packages/features/ee/workflows/components/WorkflowStepContainer.tsx

```diff
@@ -433,7 +442,7 @@ export default function WorkflowStepContainer(props: WorkflowStepProps) {
                     </div>
                   ) : (
                     <>
-                      <div className="mt-3 flex">
+                      <div className="flex mt-3">
                         <TextField
```

Comment:
https://capture.dropbox.com/RvWIJIMlQWgNhF1j
I see 2 verification boxes

Reply:
Thank you for spotting that, don't know how that happened 🤦🏼‍♀️
The first box was the old one without the team changes, I think that's why you had the issues with the verification in a team. This should be solved now